x-pack/plugins/aiops/public/components/change_point_detection/change_points_table.tsx

```diff
@@ -126,6 +167,63 @@ export const ChangePointsTable: FC<ChangePointsTableProps> = ({
             truncateText: false,
             sortable: true,
           },
+          {
+            name: i18n.translate('xpack.aiops.changePointDetection.actionsColumn', {
+              defaultMessage: 'Actions',
+            }),
+            actions: [
+              {
+                name: i18n.translate(
+                  'xpack.aiops.changePointDetection.actions.filterForValueAction',
+                  {
+                    defaultMessage: 'Filter for value',
+                  }
+                ),
+                description: i18n.translate(
+                  'xpack.aiops.changePointDetection.actions.filterForValueAction',
+                  {
+                    defaultMessage: 'Filter for value',
+                  }
+                ),
+                icon: 'plusInCircle',
+                color: 'primary',
+                type: 'icon',
+                onClick: (item) => {
+                  if (!item.group) return;
+                  filterManager.addFilters(
+                    getFilterConfig(dataView.id!, item as Required<ChangePointAnnotation>, false)!
+                  );
+                },
+                isPrimary: true,
+                'data-test-subj': 'aiopsChangePointFilterForValue',
+              },
+              {
+                name: i18n.translate(
+                  'xpack.aiops.changePointDetection.actions.filterOutValueAction',
+                  {
+                    defaultMessage: 'Filter out value',
+                  }
+                ),
+                description: i18n.translate(
+                  'xpack.aiops.changePointDetection.actions.filterOutValueAction',
+                  {
+                    defaultMessage: 'Filter out value',
+                  }
+                ),
+                icon: 'minusInCircle',
+                color: 'primary',
+                type: 'icon',
+                onClick: (item) => {
+                  if (!item.group) return;
```

Comment:
Same question as above regarding click+silent return.

Reply:
same as above, removed in https://github.com/elastic/kibana/pull/155256/commits/b85027f2bf7fcbf0bd59d595ea6c05476438eea0